content/en/docs/faq/installation/uninstall-pluggable-components.md

```diff
@@ -0,0 +1,215 @@
+
+{{</ notice >}}
+
+## Prerequisites
+
+You have to change the value of the field `enabled` from `true` to `false` in `ks-installer` before you uninstall any pluggable components except Service Topology and Pod IP Pools. For more information about how to edit `ks-installer`, see [Enable Pluggable Components](../../../pluggable-components/).
+
+## Uninstall KubeSphere App Store
+
+Change the value of `openpitrix.store.enabled` from `true` to `false` in `ks-installer`.
+
+## Uninstall KubeSphere DevOps
+
+1. Change the value of `devops.enabled` from `true` to `false` in `ks-installer`.
+
+2. Run the following commands:
+
+   ```bash
+   helm -n kubesphere-devops-system delete ks-jenkins
+   helm -n kubesphere-devops-system delete uc
+   ```
+
+   ```bash
+   # Delete DevOps projects
+   for devopsproject in `kubectl get devopsprojects -o jsonpath="{.items[*].metadata.name}"`
+   do
+     kubectl patch devopsprojects $devopsproject -p '{"metadata":{"finalizers":null}}' --type=merge
+   done
+   
+   for pip in `kubectl get pipeline -A -o jsonpath="{.items[*].metadata.name}"`
+   do
+     kubectl patch pipeline $pip -n `kubectl get pipeline -A | grep $pip | awk '{print $1}'` -p '{"metadata":{"finalizers":null}}' --type=merge
+   done
+   
+   for s2ibinaries in `kubectl get s2ibinaries -A -o jsonpath="{.items[*].metadata.name}"`
+   do
+     kubectl patch s2ibinaries $s2ibinaries -n `kubectl get s2ibinaries -A | grep $s2ibinaries | awk '{print $1}'` -p '{"metadata":{"finalizers":null}}' --type=merge
+   done
+   
+   for s2ibuilders in `kubectl get s2ibuilders -A -o jsonpath="{.items[*].metadata.name}"`
+   do
+     kubectl patch s2ibuilders $s2ibuilders -n `kubectl get s2ibuilders -A | grep $s2ibuilders | awk '{print $1}'` -p '{"metadata":{"finalizers":null}}' --type=merge
+   done
+   
+   for s2ibuildertemplates in `kubectl get s2ibuildertemplates -A -o jsonpath="{.items[*].metadata.name}"`
+   do
+     kubectl patch s2ibuildertemplates $s2ibuildertemplates -n `kubectl get s2ibuildertemplates -A | grep $s2ibuildertemplates | awk '{print $1}'` -p '{"metadata":{"finalizers":null}}' --type=merge
+   done
+   
+   for s2iruns in `kubectl get s2iruns -A -o jsonpath="{.items[*].metadata.name}"`
+   do
+     kubectl patch s2iruns $s2iruns -n `kubectl get s2iruns -A | grep $s2iruns | awk '{print $1}'` -p '{"metadata":{"finalizers":null}}' --type=merge
+   done
+   
+   kubectl delete devopsprojects --all 2>/dev/null
+   ```
+
+   ```bash
+   kubectl delete ns kubesphere-devops-system
+   ```
+
+## Uninstall KubeSphere Logging
+
+1. Change the value of `logging.enabled` from `true` to `false` in `ks-installer`.
+
+2. To disable only log collection:
+
+   ```bash
+   delete inputs.logging.kubesphere.io -n kubesphere-logging-system tail
+   ```
+
+   {{< notice note >}}
+
+   After running this command, KubeSphere stops collecting container logs.
```

Comment:
You still can view container recent logs, only you are not able to browse history logs.

Reply:
Updates made [here](https://github.com/kubesphere/website/pull/1863/commits/5fdeba6088caf26e9be3fdef124656cf6f636a22#diff-8c43bfc4100d51031fa4c69b1e26b3005862122b8f91b7410de3408072d66b0dR103).